components/style/values/specified/basic_shape.rs

```diff
@@ -642,13 +642,13 @@ pub enum GeometryBox {
     Fill,
     Stroke,
     View,
-    Shape(ShapeBox),
+    ShapeBox(ShapeBox),
```

Comment:
These too.  Please try to keep fixes in the right patches, rather than folded into the final patch. :-)